src/app/views/account/account.component.html

```diff
@@ -1,15 +1,15 @@
-<div class="row hide-sm">
+<div class="row">
   <h2 class="col-9">myTOA</h2>
   <div class="col-3">
-    <button mdc-button primary style="color: white" (click)="signOut();">{{ "pages.account.logout" | translate }}</button>
+    <button mdc-button primary (click)="signOut();">{{ "pages.account.logout" | translate }}</button>
   </div>
 </div>
 <div class="row">
   <div class="col-center">
-    <img draggable="false" class="rounded-circle img-center" src="{{profileUrl | async}}" height="100" />
-    <div style="text-align: center; font-size: 30px; font-weight: bold; color: rgb(51,51,51)">{{user['fullName']}}</div>
-    <div *ngIf="user['level'] == null" style="text-align: center; font-size: 15px; font--weight: 100; color: rgb(116, 116, 116)">{{user['team']}}</div>
-    <div *ngIf="user['level'] != null" style="text-align: center; font-size: 15px; font--weight: 100; color: rgb(116, 116, 116)">{{user['team']}} ᛫ Level {{user['level']}}</div>
+    <img draggable="false" class="rounded-circle img-center" src="{{ profileUrl | async }}" height="100" />
+    <div class="major-detail">{{ user['fullName'] }}</div>
+    <div *ngIf="user['level'] == null" class="minor-detail">{{ user['team'] }}</div>
```

Comment:
I agree for showing the team but every user "should" have a team number (of some sorts).

Reply:
That isn't true at all

Reply:
You wanted me to compress it to one line instead of two... so...

Reply:
Yes, sorry.  I was not aware that the team number form was optional.  I will add something to display team if the user is part of one.

Reply:
I added some code that shows the team if the user entered one as signup (and not show if they didn't enter one).  Basically this will hide the empty space before the dot in the middle of the line and the dot.